packages/suite/src/views/suite/SwitchDevice/DeviceItem/DeviceItem.tsx

```diff
@@ -123,6 +75,8 @@ export const DeviceItem = ({ device, instances, onCancel, backgroundRoute }: Dev
         handleRedirection();
     };
 
+    // @TODO add logic for creating standard/passphrase button
```

Comment:
Not sure if this is forgotten or some unfinished work. 

I am not really fan of TODOs in the code. I think you should:

a) finish it
b) or create issue for it and plan it
c) make peace with the fact this will never be done and just ignore it

Without the above, I have an experience that TODOs became just a *wishes* of the people long gone, polluting the code.